addon/styles/_colors.scss

```diff
@@ -56,9 +72,43 @@ $frost-color-lgrey-1: #e2e2e2 !default;
 $frost-color-lgrey-2: #dbdbdb !default;
 $frost-color-lgrey-3: #f0f2f4 !default;
 $frost-color-lgrey-4: #f5f6f7 !default;
-$frost-color-lgrey-5: #f2f2f2 !default;
+$frost-color-lgrey-5: #f2f2f2 !default;  // no longer in style guide
 
-// TODO The following three colors should be in a theme an come from the palette (talk to Brain about palette)
-$frost-color-positive: #86c43b !default;
-$frost-color-neutral: #efc059 !default;
-$frost-color-danger: #d1625e !default;
+$frost-color-white: #fff !default;
+
+// Frost status colors
+
+$frost-color-active-1: #00aa3e !default;
+$frost-color-active-2: #006823 !default;
+$frost-color-active-3: #c6ff8a !default;
+
+$frost-color-minor-1: #ddd326 !default;
+$frost-color-minor-2: #a89b00 !default;
+$frost-color-minor-3: #fff6b3 !default;
+
+$frost-color-major-1: #ef8e00 !default;
+$frost-color-major-2: #ba6a00 !default;
+$frost-color-major-3: #fdc !default;
+
+$frost-color-critical-1: #e52c2c !default;
+$frost-color-critical-2: #a31414 !default;
+$frost-color-critical-3: #ffbdbd !default;
+
+$frost-color-warn-1: #7d85b2 !default;
+$frost-color-warn-2: #36477f !default;
+$frost-color-warn-3: #cacfe5 !default;
+
+$frost-color-loa-1: #00f2f2 !default;  // Used for LoA alarms
+$frost-color-loa-2: #2e5e5e !default;
+$frost-color-loa-3: #abfffb !default;
+
+// Alarm colors
+
+$frost-color-clear: $frost-color-white;  // cleared alarm
+$frost-color-info-1: #b9cfea !default;
+$frost-color-warn-1: #7d8fb2 !default;
+$frost-color-minor-1: #ddd326 !default;
+$frost-color-major-1: #ef8e00 !default;
+$frost-color-critical-1: #e52c2c !default;
+// Loss of Association alarms should use loa-1
```

Comment:
I am not sure what the comment means? Should this be LOA1: `#00F2F2`?

Reply:
Ah, I understand now. Maybe the comment should instead say,

```
// Loss of Association alarms uses $frost-color-loa-1`

Reply:
I like your wording a lot better :-)